evalml/preprocessing/data_splitters/base_splitters.py

```diff
@@ -0,0 +1,106 @@
+from sklearn.model_selection import StratifiedKFold, train_test_split
+from sklearn.model_selection._split import BaseCrossValidator
+
+from evalml.utils import _convert_numeric_dataset
+
+
+class BaseTVSplit(BaseCrossValidator):
+    """Base class for training validation data splitter."""
+
+    def __init__(self, sampler=None, test_size=None, random_seed=0):
+        """Create a training-validation data splitter instance
+
+        Arguments:
+            sampler (sampler instance): The sampler instance to use for resampling the training data. Must have a `fit_resample` method. Defaults to None.
+
+            test_size (float): What percentage of data points should be included in the validation
+                set. Defalts to the complement of `train_size` if `train_size` is set, and 0.25 otherwise.
+
+            random_seed (int): Random seed for the splitter. Defaults to 0
+        """
+        self.sampler = sampler
+        self.test_size = test_size
+        self.random_seed = random_seed
+
+    @staticmethod
+    def get_n_splits():
+        """Returns the number of splits of this object."""
+        return 1
+
+    def split(self, X, y):
+        """Splits and returns the sampled training data using the data sampler provided.
+
+        Arguments:
+                X (ww.DataTable): DataTable of points to split
+                y (ww.DataTable): DataColumn of points to split
+
+        Returns:
+            tuple((pd.DataFrame, pd.Series), (pd.DataFrame, pd.Series)): A tuple containing the resulting ((X_train, y_train), (X_test, y_test)) post-transformation.
+        """
+        X, y = _convert_numeric_dataset(X, y)
+        X_train, X_test, y_train, y_test = train_test_split(X, y, test_size=self.test_size, random_state=self.random_seed)
```

Comment:
It defaulted to returning pandas instead of woodwork. Since we no longer need to return woodwork datatables post-split, I can remove the support for returning woodwork in that util.